syntax/Jenkinsfile.vim

```diff
@@ -3,11 +3,11 @@ syn keyword jenkinsfileBuiltInVariable currentBuild
 
 syn keyword jenkinsfileSection pipeline agent stages steps
 
-syn keyword jenkinsfileDirective environment options parameters triggers stage tools input when
+syn keyword jenkinsfileDirective environment options parameters triggers stage tools input when libraries
 
 syn keyword jenkinsfileOption contained buildDiscarder disableConcurrentBuilds overrideIndexTriggers skipDefaultCheckout nextgroup=jenkinsfileOptionParams
 syn keyword jenkinsfileOption contained skipStagesAfterUnstable checkoutToSubdirectory timeout retry timestamps nextgroup=jenkinsfileOptionParams
-syn region  jenkinsfileOptionParams contained start="(" end=")" transparent contains=@groovyTop
+syn region  jenkinsfileOptionParams contained start='(' end=')' transparent contains=@groovyTop
```

Comment:
For my information, why change the double quotes to single quotes?

Reply:
It's a general best practice in Vim Script.

Because the `"` (double quote) is the comment-start character.

Reply:
Ah, of course... thanks!